evalml/objectives/fraud_cost.py

```diff
@@ -36,23 +36,6 @@ def __init__(
         self.fraud_payout_percentage = fraud_payout_percentage
         self.amount_col = amount_col
 
-    def decision_function(self, ypred_proba, threshold=0.0, X=None):
-        """Determine if a transaction is fraud given predicted probabilities, threshold, and dataframe with transaction amount.
-
-        Arguments:
-            ypred_proba (pd.Series): Predicted probablities
-            threshold (float): Dollar threshold to determine if transaction is fraud
```

Comment:
This `threshold` parameter didn't really make sense to me. For our fraud prediction models, we are predicting whether or not a transaction is fraud, so the `predicted_proba` output should be a probability on whether the transaction was fraud or not. Multiplying this probability by the amount spent shouldn't be helpful in determine if there was fraud or not, so I don't think we should threshold based on some spent amount. 

I removed this so that we use the probabilities like our other objectives. Let me know if I'm misinterpreting!

Reply:
I guess the idea was to label as fraud the transactions whose expected loss due to fraud was larger than some threshold.

Maybe this would be helpful in prioritizing larger transactions?

I'm just thinking out loud. I think this is ok from my point of view.

Reply:
Yeah, that seemed reasonable. This also definitely didn't work well with the `Bounded` method, since we bound search from [0, 1], but it also didn't work well with `Brent` and `Golden` methods since those resulting thresholds generally stayed <2, and oftentimes went negative! 

It makes sense to prioritize larger transactions, but that should hopefully be something the model can find rather than relying on the objective, imo